website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -4,391 +4,275 @@ linkTitle: Releasing
-...
-
-### Deprecated
-- [Short description of feature deprecated] (#PR)
-...
+Lock down the `master` branch to prevent further commits before the release completes.
+See [this section](#locking-down-branches) for steps to do so.
 
-### Removed
-- [Short description of feature removed] (#PR)
-...
+### 3. Create and merge a new PR
 
-### Bug Fixes
-- [Short description of bug and fix] (#PR)
-...
-```
+Create and merge a new PR for the above commit. You can force-merge your PR to the locked-down `master`
+if you have admin access to the operator-sdk repo, or ask an administrator to do so.
 
-## Release Signing
+### 4. Unlock the `master` branch
 
-When a new release is created, the tag for the commit it signed with a maintainers' gpg key and
-the binaries for the release are also signed by the same key. All keys used by maintainers will
-be available via public PGP keyservers such as pool.sks-keyservers.net.
+See [this guide](#unlocking-branches) for steps to do so.
 
-For new maintainers who have not done a release and do not have their PGP key on a public
-keyserver, output your armored public key using this command:
+### 5. Create and push a release tag
 
 ```sh
-$ gpg --armor --export "$GPG_EMAIL" > mykey.asc
+make tag RELEASE_VERSION=v1.3.0
+git push upstream v1.3.0
 ```
 
-Then, copy and paste the content of the outputted file into the `Submit a key` section on
-pool.sks-keyservers.net or any other public keyserver that synchronizes
-the key to other public keyservers. Once that is done, other people can download your public
-key and you are ready to sign releases.
-
-## Verifying a release
+### 6. Fast-forward the `latest` and release branches
 
-To verify a git tag, use this command:
+The `latest` branch points to the latest release tag to keep the main website subdomain up-to-date.
+Run the following commands to do so:
 
 ```sh
-$ git verify-tag --verbose "$TAG_NAME"
+git checkout latest
+git reset --hard tags/v1.3.0
+git push -f upstream latest
 ```
 
-If you do not have the mantainers public key on your machine, you will get an error message similiar to this:
-
-```console
-$ git verify-tag --verbose "$TAG_NAME"
-object 61e0c23e9d2e217f8d95ac104a8f2545c102b5c3
-type commit
-tag v0.6.0
-tagger Ish Shah <ishah@redhat.com> 1552688145 -0700
-
-Operator SDK v0.6.0
-gpg: Signature made Fri Mar 15 23:15:45 2019 CET
-gpg:                using RSA key <KEY_ID>
-gpg: Can't check signature: No public key
-```
-
-To download the key, use the following command, replacing `$KEY_ID` with the RSA key string provided in the output of the previous command:
+Similarly, to update the release branch, run:
 
 ```sh
-$ gpg --recv-key "$KEY_ID"
+git checkout v1.3.x
+git reset --hard tags/v1.3.0
+git push -f upstream v1.3.x
 ```
 
-To verify a release binary using the provided asc files see the [installation guide.][install-guide]
+### 7. Post release steps
 
-## Release steps
+See the [post-release section](#post-release-steps).
```

Comment:
Again I'd put the post release steps inline with this section. And I'd probably either number them or bullet them:

* Announce the release
   * Send an email to the mailing list. Post to Kubernetes slack in #kubernetes-operators and #operator-sdk-dev.
* Bump open issues to the next release
   * In the GitHub milestone, bump any open issues to the following release.